documentation/js-sdk-guide/modules/ROOT/pages/getting-started-guide.adoc

```diff
@@ -0,0 +1,41 @@
+////
+Copyright (c) 2023 Robert Bosch Manufacturing Solutions GmbH
+
+See the AUTHORS file(s) distributed with this work for additional information regarding authorship.
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
+If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/
+SPDX-License-Identifier: MPL-2.0
+////
+
+= Getting Started Guide
+
+== Set up local environment
+
+=== Prerequisites
+
+Install https://nodejs.org/en/download/[node] LTS (long term support) version.
+
+=== Install SAMM Schematic
+
+Run the following command in the desired Angular project folder to add the dependency
+
+[source]
+npm install https://github.com/eclipse-esmf/esmf-sdk-js-schematics/releases/download/vX.X.X/esmf-semantic-ui-schematics-X.X.X.tgz --save-dev
```

Comment:
Can you use a version macro to use an actual released version, e.g.
`npm install .../download/{esmf-sdk-js-schematics-version}/...`
(And have the value set in site.yml in the root docs project)